extensions/kafka/src/main/java/io/deephaven/kafka/publish/GenericRecordKeyOrValueSerializer.java

```diff
@@ -305,8 +307,10 @@ private static GenericRecordFieldProcessor makeInstantToMicrosFieldProcessor(
         return makeGenericFieldProcessor(
                 fieldName,
                 chunkSource,
-                (final int ii, final ObjectChunk<?, Values> inputChunk) -> DateTimeUtils
-                        .epochMicros((Instant) inputChunk.get(ii)));
+                (final int ii, final ObjectChunk<?, Values> inputChunk) -> {
+                    Instant val = (Instant) inputChunk.get(ii);
+                    return val == null ? null : DateTimeUtils.epochMicros(val);
+                });
```

Comment:
We explicitly want to return `null : Long` as opposed to `NULL_LONG`?

Otherwise, `io.deephaven.time.DateTimeUtils#epochMicros(java.time.Instant)`